account_invoice_refund_reason/__manifest__.py

```diff
@@ -0,0 +1,21 @@
+# Copyright (C) 2019 - TODAY, Open Source Integrators
+# Copyright (C) 2019 Serpent Consulting Services Pvt. Ltd.
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    'name': 'Account Invoice Refund Reason',
+    'version': '12.0.1.0.0',
+    "summary": "Account Invoice Refund Reason.",
+    'category': 'Accounting',
+    'author': "Open Source Integrators, "
```

Comment:
@Chanakya-SerpentCS @max3903 Please add `Odoo Community Association (OCA)` in author part